ISISScientificComputing/autoreduce
# Use exisitng queue client in webapp

### Summary of work
* Get webapp to import `QueueClient` from `utils` package
* There are no unit tests to cover this due to a lack of framework for webapp tests currently
* Remove queue_processor class completely from webapp

### How to test your work
* Go to the development web app
* Resubmit a run via the webapp
* Ensure that the run successfully re-runs

### Additional comments
* This is the last issue to allow #517 to close

Fixes #517 


**Before merging ensure the release notes have been updated**
